vscode_client_2/README.md

```diff
@@ -1,6 +1,8 @@
 # Mainframe Messenger for VS Code
+Mainframe messenger is an easy way to send messages from Visual Studio Code to another mainframe user. 
```

Comment:
Good and concise

Reply:
thank you. Forgot to compliment your too :D
🗡️